src/org/openstreetmap/josm/plugins/osmobjinfo/OSMObjInfoActions.java

```diff
@@ -61,29 +74,93 @@ public static void openinBrowserChangeset(String idChangeset) {
 
     public static void openinBrowserChangesetMap(String idChangeset) {
         if (!idChangeset.isEmpty()) {
-            openInBrowser("https://osmcha.mapbox.com/".concat(idChangeset));
+            openInBrowser("https://osmcha.org/".concat(idChangeset));
         }
     }
 
-    public static void copyIdobj(String typeObj, String idobj) {
-        if (typeObj != null && !idobj.isEmpty()) {
-            String linkobjid = "https://www.openstreetmap.org/" + typeObj + "/" + idobj;
-            StringSelection selection = new StringSelection(linkobjid);
-            Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
-            clipboard.setContents(selection, selection);
-            new Notification(tr("Copy: " + linkobjid)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
+    public static void copyIdobj(List<AllOsmObjInfo> osmObjInfo) {
+        if (osmObjInfo == null || osmObjInfo.isEmpty()) return;
+        String linkobjid = "";
+        StringBuilder builder = new StringBuilder();
+        for (AllOsmObjInfo object : osmObjInfo) {
+            builder.append("https://www.openstreetmap.org/")
```

Comment:
Config.getUrls().getBaseBrowseUrl() should be used instead of hardcoding https://www.openstreetmap.org